pkg/util/util.go

```diff
@@ -391,3 +391,17 @@ func EncodeURL(href string) (string, error) {
 	u.RawQuery = u.Query().Encode()
 	return u.RequestURI(), nil
 }
+
+func RemoveDuplicateStr(strSlice []string) []string {
+	allKeys := make(map[string]bool)
+	var list []string
+	for _, item := range strSlice {
+		if _, value := allKeys[item]; !value {
+			allKeys[item] = true
+			if len(item) > 0 {
+				list = append(list, item)
+			}
+		}
+	}
+	return list
```

Comment:
do we want these comma-seperated items sorted?

Reply:
I have checked response, and the order of array elements is same, which gives labels in same order.

Reply:
This list is child object names and they don't need to be sorted.